01_install_requirements.sh

```diff
@@ -35,13 +35,6 @@ sudo dnf -y upgrade
 source /etc/os-release
 export DISTRO="${ID}${VERSION_ID%.*}"
 if [[ $DISTRO == "centos8" ]]; then
-    if [ "$NAME" != "CentOS Stream" -a ! -e /etc/yum.repos.d/centos-vault.repo ] ; then
-        # Centos8 is EOL
-        mv /etc/yum.repos.d /etc/yum.repos.d_$(date +%s)
```

Comment:
What about adding a deprecation message in such case, ie "CentOS is not supported anymore"?

Reply:
sounds good. added